builder/vmware/common/driver_config.go

```diff
@@ -44,3 +52,50 @@ func (c *DriverConfig) Prepare(ctx *interpolate.Context) []error {
 
 	return nil
 }
+
+func (c *DriverConfig) Validate(SkipExport bool) error {
+	if c.RemoteType == "esx5" && SkipExport != true {
```

Comment:
Here I would make these ifs guard closes insteal, so that the main code of this func is on the leftmost level.

Reply:
Ah, I like this idea; pushed. I also considered also moving up the SkipValidateCredentials check the same line as `if c.RemoteType == "esx5" && SkipExport != true {` but I want to make sure the password is not empty regardless of the value of SkipValidateCredentials.